Docs/doc.css

```diff
@@ -0,0 +1,144 @@
+    @import url('../CSS/reset.css');
+    @import url('../CSS/utility.css');
+    @import url('../CSS/variable.css');
+    @import url('../CSS/nav.css');
+    @import url('../CSS/footer.css');
+    .color-text {
+        color: var(--prime-blue);
+        font-weight: 600;
+    }
+    
+    .flex-row {
+        width: 90%;
+        margin: 0 auto;
+    }
+    
+    .menuList {
+        padding: 3rem 0 0 3rem;
+        flex-basis: 250px;
+        min-width: 250px;
+        height: 100vh;
+        font-size: 1.3rem;
+    }
+    
+    .active {
+        color: var(--accent-color);
+        font-weight: bolder;
+        padding-left: 1rem;
+    }
+    
+    .comp-head {
+        font-weight: bolder;
+    }
+    
+    .comp-el a:hover {
+        color: var(--accent-color);
+        font-weight: 700;
+        padding-left: 1rem;
+    }
+    
+    .main-content {
+        flex-grow: 1;
+        /* background-color: red; */
+    }
+    
+    #main-heading {
+        font-size: 3rem;
+    }
+    
+    .sub-heading {
+        font-size: 2rem;
+    }
+    
+    .headings {
+        font-size: 2.5rem;
+    }
+    
+    p,
+    .bottom-nav {
+        font-size: medium;
+        letter-spacing: 0.8px;
+    }
+    
+    .divider {
+        border-top: 1px solid grey;
+    }
+    
+    .element-display {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-wrap: wrap;
+        border: 1px solid;
+        border-radius: 10px;
+        padding: 2rem;
+        gap: 10px;
+    }
+    
+    .color {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+        height: 10rem;
+        width: 10rem;
+        font-size: small;
+        letter-spacing: 1px;
+        font-weight: 600;
+        padding: 1rem;
+    }
+    
+    .primary {
+        background-color: #1877f2;
+    }
+    
+    .seccondary {
+        background-color: #fefdff;
+    }
+    
+    .accent {
+        background-color: #14bdeb;
+    }
+    
+    .danger {
+        background-color: #F44336;
+    }
+    
+    .success {
+        background-color: #4CAF50;
+    }
+    
+    .warning {
```

Comment:
You can make variables for these colors as they are gonna be used twice or thrice in whole library.